bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -67,14 +67,15 @@ <h1>{{ self.page_title() }}</h1>
     ] -%}
     {% include promos|random() %}
   {% else %}
-    {% call hero(
-      title='Firefox products are designed to protect your privacy',
-      class='privacy-promise-hero mzp-has-image mzp-t-dark mzp-t-firefox',
+    {% call split(
       image_url='img/firefox/privacy/promise/privacy-hero.png',
       include_highres_image=True,
-      include_cta=True,
-      heading_level=2
+      media_after=True,
```

Comment:
Similar suggestion for `mzp-l-split-hide-media-on-sm-md` - live version has no image on smaller screens and I'm inclined to less scrolling on mobile where possible.